paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -168,6 +170,29 @@ internal class DefaultFlowController @Inject internal constructor(
         configuration: PaymentSheet.Configuration?,
         callback: PaymentSheet.FlowController.ConfigCallback
     ) {
+        try {
+            configuration?.validate()
+            if (clientSecret.value.isBlank()) {
+                if (clientSecret is PaymentIntentClientSecret) {
+                    throw InvalidParameterException(
+                        "The `paymentIntentClientSecret` passed to `configureWithPaymentIntent` " +
+                            "cannot be an empty string."
+                    )
+                } else if (clientSecret is SetupIntentClientSecret) {
+                    throw InvalidParameterException(
+                        "The `setupIntentClientSecret` passed to` configureWithSetupIntent` " +
+                            "cannot be an empty string."
+                    )
+                }
+            }
```

Comment:
are we missing an exception for 
```throw InvalidParameterException("Client secret cannot be blank.")```

like we do above, inside the `if clientSecret.value.isBlank()` case?